Scripts/script-ADGetUser.yml

```diff
@@ -90,8 +90,12 @@ script: |-
               if 'UserAccountControl' not in m['Contents'][0] or m['Contents'][0]['UserAccountControl'] == "":
                   continue
               if userAccountControlOut:
+                  accountControlFlags = ""
```

Comment:
I would do something like this instead of string manipulation:

```
              if userAccountControlOut:
                  accountControlFlags = []
                  for flag, mask in UserAccountControlFlags.iteritems():
                      if int(m['Contents'][0]['UserAccountControl'],10) & mask != 0:
                          accountControlFlags.append(flag)
                  m['Contents'][0]['UserAccountControl'] = ', '.join(accountControlFlags)
```

Reply:
@dorsha yeah looks better, thanks!